projects/liquidium/index.js

```
@@ -0,0 +1,144 @@
+  const meta = ASSET_META[asset]
+  if (!meta) return
+
+  const borrowed = toNumber(pool.total_debt_at_last_sync) / 10 ** meta.decimals
+  if (borrowed > 0) api.addCGToken(meta.coingeckoId, borrowed)
+}
+
+async function chainTvl(api, chain) {
+  const pools = await getPools()
+  pools
+    .filter((pool) => CHAIN_MAP[getVariantKey(pool.chain)] === chain)
+    .forEach((pool) => addNetSupply(api, pool))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)